oak/server/oak_node.cc

```diff
@@ -183,7 +186,15 @@ static void RunModule(wabt::interp::Environment* env, wabt::interp::DefinedModul
   }
 }
 
-OakNode::OakNode() : Service() {}
+OakNode::OakNode() : Service() {
+  std::unique_ptr<ChannelHalf> storage_read_channel =
```

Comment:
Should this be around

https://github.com/project-oak/oak/blob/7d75624bd36812d3f7c6bef09b70402e0354190e/oak/server/oak_node.cc#L188-L263

?

cc @daviddrysdale

Reply:
Yeah, keeping all of the well-known channel creation stuff together makes sense, either here or in `CreateNode` (and I'd probably go for the latter because we don't want to start a logging thread until we're sure the Node instance is good to go).

Reply:
Done.